floris/tools/optimization/yaw_optimization/yaw_optimization_base.py

```diff
@@ -533,23 +549,24 @@ def _finalize(self, farm_power_opt_subset=None, yaw_angles_opt_subset=None):
         ti = np.min(self.fi.floris.flow_field.turbulence_intensities)
         df_list = []
         num_wind_directions = len(self.fi.floris.flow_field.wind_directions)
-        for ii, wind_speed in enumerate(self.fi.floris.flow_field.wind_speeds):
-            df_list.append(
-                pd.DataFrame(
-                    {
-                        "wind_direction": self.fi.floris.flow_field.wind_directions,
-                        "wind_speed": wind_speed * np.ones(num_wind_directions),
-                        "turbulence_intensities": ti * np.ones(num_wind_directions),
-                        "yaw_angles_opt": list(self.yaw_angles_opt[:, ii, :]),
-                        "farm_power_opt": None
-                        if self.farm_power_opt is None
-                        else self.farm_power_opt[:, ii],
-                        "farm_power_baseline": None
-                        if self.farm_power_baseline is None
-                        else self.farm_power_baseline[:, ii],
-                    }
-                )
+        wd_array = self.fi.floris.flow_field.wind_directions
+        ws_array = self.fi.floris.flow_field.wind_speeds * np.ones(num_wind_directions)
+        df_list.append(
+            pd.DataFrame(
+                {
+                    "wind_direction": wd_array,
+                    "wind_speed": ws_array,
+                    "turbulence_intensity": ti * np.ones(num_wind_directions),
```

Comment:
similarly, since `self.fi.floris.flow_field.turbulence_intensities` is an array now, I think this should just be `self.fi.floris.flow_field.turbulence_intensities`.